build.gradle

```diff
@@ -255,8 +277,31 @@ dependencies {
     rewrite("org.openrewrite.recipe:rewrite-logging-frameworks")
     rewrite("org.openrewrite.recipe:rewrite-testing-frameworks")
     rewrite("org.openrewrite.recipe:rewrite-migrate-java")
+
+
+}
+
+// simple tool configuration based on https://github.com/spotbugs/spotbugs-gradle-plugin/blob/master/README.md
```

Comment:
You can remove that line, because after short google one finds that.

The keys are self-explanatorey.